tests/integration/test_workflow_db_mgr.py

```diff
@@ -178,3 +179,36 @@ async def test_record_only_non_clock_triggers(
     assert db_select(schd, False, 'xtriggers', 'signature') == [
         ('xrandom(100)',),
         ('xrandom(100, _=Not a real wall clock trigger)',)]
+
+
+async def test_time_zone_writing(
+    one_conf,
+    flow,
+    scheduler,
+    run,
+    complete,
+    set_nonexistent_timezone,
+    db_select,
+):
+    """Don't store scheduler startup timezone forever.
+
+    https://github.com/cylc/cylc-flow/issues/6701
+    """
+    wid = flow(one_conf)
+    schd = scheduler(wid, paused_start=False, run_mode='live')
+    async with run(schd):
+        await complete(schd, timeout=20)
```

Comment:
Unfortunately the integration test running in live mode is falling foul of MacOS runner performance, looks like the job is taking more than 20s to run. But actually we don't need the job or workflow to run, can instead just use `async with start(schd)` and prod the task through the submission process manually by `schd.submit_task_jobs()` or something.